### DISCUSSION TITLE
✨ [Feature]: 新增Dashboard组件

### DISCUSSION THREAD

**Discussion by kagol** [DevCloudFE/vue-devui#1403] [General]
### What problem does this feature solve

参考：
https://devui.design/components/zh-cn/dashboard/demo

### What does the proposed API look like

参考：
https://devui.design/components/zh-cn/dashboard/demo

**Comment by xzxldl55** [comment]
认领 dashboard 可以吗

**Comment by opendesign-os** [comment]
I am currently building a framework-independent Dashboard based on GridStack, with wrappers for Vue and React. Is the project still actively maintained? I would be happy to contribute my Dashboard if needed.
